openai_files/helpers.py

```diff
@@ -10,8 +10,18 @@ def get_system_message(prompt_type: PromptType) -> str:
 
 
 PROMPTS = {
+    PromptType.GENERATE_IMAGE_HAPPY: lambda holiday_list: (
+        f"Here is a list of holidays: {holiday_list}\n"
+        "Please create an image that combines all these events with the context of a happy crypto "
+        "investor as the subject."
+    ),
+    PromptType.GENERATE_IMAGE_SAD: lambda holiday_list: (
+        f"Here is a list of holidays: {holiday_list}\n"
+        "Please create an image that combines all these events with the context of a sad crypto "
+        "investor as the subject, but make it a little bit funny."
+    ),
     PromptType.GET_HOLIDAYS: lambda date: (
-        f"List all international commemorations, including national holidays, observances, "
+        "List all international commemorations, including national holidays, observances, "
```

Comment:
i think we've worked with these before but they're just f-strings. you can coerce variables into strings and place them within strings like `f"hello {name}"`. This is a per-line settings so if you have multi-line strings like this prompt, we should only place an `f` if we need them like we do on the following line. I just started this as a single-line f-string since i knew i'd add the date in somewhere, but forgot to remove the `f` on the first line.